Automattic/portkey-android
# Bring upload icon back to trigger the media picker

Fix #617 as per this https://github.com/Automattic/stories-android/issues/617#issuecomment-738164652

> > The thumbnail was removed long time ago but I think we can get it back for this specific case as a quick workaround for v1

> That sgtm

This PR brings the upload icon to the camera preview screen back, so there's a way to opt to open the media picker when editing an empty story block from Gutenberg.

Technically this pretty much just reverts this PR https://github.com/Automattic/stories-android/pull/447.

![upload_icon](https://user-images.githubusercontent.com/6597771/101173502-64ece700-3621-11eb-84a0-23013fe678c4.gif)


**To test**
1. use WPAndroid PR  https://github.com/wordpress-mobile/WordPress-Android/pull/13543
2. create a gutenberg blog post
3. add a Story block from the picker
4. tap on "ADD MEDIA" on the empty Story block
5. observe the camera preview mode of the Story composer appears, and the Upload thumbnail icon appears to the left of the camera shutter
6. tap on it, and the media picker should display 🎉 

Try several mixes of adding new captures, picked pictures, etc, to verify it works.
